app/views/finders/_applied_filter.mustache

```diff
@@ -0,0 +1,18 @@
+<div class="facet-tag-group">
+{{#.}}
+	<div class="facet-tag-wrapper">
```

Comment:
Nitpick: indentation on this file seems too big?

Reply:
Yeah, this is strange... it looks ok in my editor (which uses the correct spacing for every other file). I've tried re-indenting but it doesn't show up as a change, so my editor at least thinks it's correct.

Reply:
Creating a new mustache file and deleting the old one has fixed it.